layouts/shortcodes/release-branches.html

```diff
@@ -0,0 +1,45 @@
+<h2 id="detailed-release-history-for-active-branches">Detailed Release History for Active Branches</h2>
```

Comment:
I would leave this out from the shortcode. Let localization teams write:
```markdown
## Detailed release history for active branches
```

or whatever they wish to in their target localization.